docs/start_nominating.md

```diff
@@ -125,4 +125,10 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
   <img src={useBaseUrl('/nominator-guide/nominate-validator-4.png')} />
 </div>
 
+:::note
+
+Make sure to revisit your nominations once in a while. If some of your validators decide to change their commission percentage, your respective nominations will be removed and you might need to place new nominations.
```

Comment:
```suggestion
Make sure to revisit your nominations once in a while. The commission is subject to change without notice, which is likely to happen if you nominate validators with 0% commission. 
```

We deep-dived into this topic and it is really true that a validator can go from 0 to 100% commission from one era to another. 
Refs:
https://discord.com/channels/803927962709131274/823238869008777267/824651028300431370
https://github.com/paritytech/substrate/issues/6920

Reply:
Thanks for your comment. Does this mean that changing the commission percentage will not invalidate/remove all previous nominations?

Reply:
yes, sadly 👎